CHANGELOG.md

```diff
@@ -4,11 +4,19 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
-## Unreleased
+## 2.4.1 (2021-08-19)
 
+This release contains bug fixes since the 2.4.0 release.
+
+We deem it high priority to upgrade since it is needed to support
+PostgreSQL 12.8 and 13.4.
```

Comment:
```suggestion
This release contains bug fixes since the 2.4.0 release.
We deem it high priority to upgrade.
The release fixes continous aggregate refresh for postgres 12.8 and 13.4,
a crash with alter table commands and a crash with continuous aggregates with HAVING clause.
```